ra2ce/network/network_config_data/enums/aggregate_wl_enum.py

```diff
@@ -7,3 +7,6 @@ class AggregateWlEnum(Ra2ceEnumBase):
     MAX = 2
     MEAN = 3
     INVALID = 99
+
+    def get_suffix(self):
```

Comment:
Although I like the technique I don't quite see  how this is logical, are we just using `MAX, MEA, INV` ? in that case it would be a `get_prefix`, not `_suffix`, a better approach would be `get_wl_file_name`. Also, add docstring explaining what is this method for.